dev-python/colormath2/colormath2-3.0.3.ebuild

```
@@ -0,0 +1,24 @@
+# Copyright 1999-2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+DISTUTILS_USE_PEP517=setuptools
+PYTHON_COMPAT=( python3_{10..13} )
+
+inherit distutils-r1
+
+DESCRIPTION="Easy color scales and color conversion for Python"
+HOMEPAGE="http://python-colormath.readthedocs.org/"
+SRC_URI="https://github.com/bkmgit/python-colormath2/archive/${PV}.tar.gz -> ${P}.gh.tar.gz"
+
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="~amd64"
+
+S="${WORKDIR}/python-colormath2-${PV}"
+
+RDEPEND="dev-python/networkx[${PYTHON_USEDEP}]
+	dev-python/numpy[${PYTHON_USEDEP}]"
```

```suggestion
RDEPEND="
	dev-python/networkx[${PYTHON_USEDEP}]
	dev-python/numpy[${PYTHON_USEDEP}]
"
```

please move `S` directly below `SRC_URI`.

---

Also fixed